### ISSUE TITLE
Webhook event : Disable retries with the Sanity Webhook event

### ISSUE THREAD

**Issue by SandeepKumarYaramchitti** [sanity-io/sanity#4132]
Hi Team, 

I am currently using AWS API gateway to receive the event from sanity webhook, then process the data in my backend application. 

However, i am seeing couple of issues and would be great if someone can help on this. 

 For successful webhook events, sanity is showing resultCode with 500. However, I am able to receive the event and process it successfully. So how do we set the resultCode to 200 so it does not reprocess. 

Alternatively, do we have a way to stop the retries because of the retries - it is adding up my AWS run times / cost. 

```
{
"id": "atm-2KpiA0tOvqq7lAKQn8b2wEnvbLb",
"projectId": "p2xc0nyv",
"inProgress": false,
"duration": 283,
"createdAt": "2023-01-25T20:16:17.210Z",
"updatedAt": "2023-01-25T20:16:17.210Z",
"messageId": "msg-2KphIlH5nBzHGUWXbl7HpqQYEQW",
"hookId": "DYfauozrrkTKhXCB",
"isFailure": true,
"failureReason": "other",
"resultCode": 502,
"resultBody": "{\"message\": \"Internal server error\"}"
}
```

**Comment by SandeepKumarYaramchitti** [comment]
I was able to do this by sending a correct response code from API gateway!. 

However, It would be great to add an option for user to configure (optional) if they want retry or not.

Keeping it open for now, but i am all set with the above issue.

**Comment by Designer023** [comment]
Yeah I have just run into this issue, I am trying to debug a revalidate API from the webhook, but it keeps sending old hook responses, presumably from a retry. I am just adding a `res.status(200).json({ ok: true });` in the hope that it clears them out.

**Comment by bjoerge** [comment]
Closing as stale as I don't think there's anything actionable from our side here, and it looks like the question has been answered. Feel free to open a new issue if this is still unclear/unanswered.
